File: output/ChatGPT/codenet/Java/Python/s059916837.py
def find(a, b):
    calc = 0
    for x in range(len(a)):
        if a[x] != b[x]:
            calc += 1
    return calc

S = input()
T = input()
min_val = float('inf')

for i in range(len(S) - len(T) + 1):
    min_val = min(min_val, find(S[i:i+len(T)], T))

if min_val != float('inf'):
    print(min_val)
else:
    print(len(T))

File: output/ChatGPT/codenet/Java/C++/s059916837.cpp
#include <iostream>
#include <string>

using namespace std;

int find(string a, string b){
    int calc = 0;
    for(int x = 0; x<a.length(); x++){
        if(a[x] != b[x]){
            calc++;
        }
    }
    return calc;
}

int main() {
    string S, T;
    cin >> S >> T;
    int min = INT_MAX;
    for(int i = 0; i<=S.length()-T.length(); i++){
        min = min(min, find(S.substr(i, T.length()), T));
    }
    if(min!=INT_MAX){
        cout << min << endl;
    }else{
        cout << T.length() << endl;
    }


    return 0;
}